Automattic/vip-go-mu-plugins
# Update Jetpack 11.1 to 11.1.1

## Description
This updates Jetpack 11.1 to 11.1.1 which includes a Dashboard bugfix: avoid JavaScript errors and layout problems on sites experiencing URL-related connection issues with WordPress.com.

## Changelog Description

### Plugin Updated: Jetpack 11.1

We upgraded Jetpack 11.1 to Jetpack 11.1.1.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Check out PR
2) Use an unpinned local site
3) Go to Jetpack > Settings and look for 11.1.1 as the default version
